tensorboard/plugins/projector/vz_projector/vz-projector-inspector-panel.ts

```diff
@@ -25,9 +25,18 @@ namespace vz_projector {
       metadataColumn: String,
       numNN: {type: Number, value: 100},
       updateNumNN: Object,
+      spriteMetadata: Object,
```

Comment:
Should this property be `spriteMeta`?

(Or should the field be `spriteMetadata`?)